packages/js/settings-editor/src/components/sidebar/index.tsx

```diff
@@ -19,15 +19,15 @@ const { Icon, ...icons } = IconPackage;
 
 const SidebarNavigationScreenContent = ( {
 	activePage,
-	settingsData,
+	pages,
 }: {
 	activePage: string;
-	settingsData: SettingsData;
+	pages: { [ key: string ]: SettingsPage };
```

Comment:
A minor suggestion: It might be worth it to create a new type for the pages for reuse in other places.